server/app.js

```diff
@@ -13,11 +13,13 @@ import projectRouter from "./routes/api/projects";
 import relationshipRouter from "./routes/api/relationships";
 
 var app = express();
+var bodyParser = require("body-parser");
 
 // cors
 app.use(cors());
 
-app.use(bodyParser.json({ limit: '10mb' }))
+// What is this?
```

Comment:
Its to limit the size of file uploads to the backend.

We might want to add the limit to the front-end too, as right now users don't know if an error happened on the backend (such as during file upload)

Reply:
Ah I see, I can add the validations to the front end any time